src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -824,6 +828,10 @@ private void DetectAttributeAndOptionConflicts(DiagnosticBag diagnostics)
                     }
                 }
             }
+            if (_compilation.Options.PublicSign && (object)this.AssemblyKeyFileAttributeSetting != (object)CommonAssemblyWellKnownAttributeData.StringMissingValue)
```

Comment:
Yeah, this seems like personal preference. I like it at the bottom, but that's just me. You can structure it how you like.

---
In reply to: [63419422](https://github.com/dotnet/roslyn/pull/11334#discussion_r63419422) [](ancestors = 63419422)